spec/index.bs

```diff
@@ -3677,24 +3701,41 @@ GPUQueue includes GPUObjectBase;
 </script>
 
 <dl dfn-type="method" dfn-for="GPUQueue">
-    : <dfn>writeBuffer(sourceData, sourceOffset, destination, destinationOffset, size)</dfn>
+    : <dfn>writeBuffer(|sourceData|, |sourceOffset|, |destination|, |destinationOffset|, |size|)</dfn>
```

Comment:
I don't think `|a|` works here (outside an algorithm)? We can chat about how this should work later.